Makefile

```diff
@@ -29,17 +29,11 @@ SCHEMA_DEFINITION_FILE := ${SIGNALS_FRONTEND_PATH}/internals/schemas/${CONFIGURA
 CONFIG_BASE_FILE := ${SIGNALS_FRONTEND_PATH}/app.amsterdam.json
 CONFIG_TEST_FILE := /tmp/app.${DOMAIN}.json
 
-ifeq ($(ENVIRONMENT),acceptance)
-SCHEMA_ENVIRONMENT := acc
-else
-SCHEMA_ENVIRONMENT := prod
-endif
-
 define _validate_schema =
 	echo validating schema - domain=$(2), environment=$(ENVIRONMENT), schema environment=${3} && \
 	npm i && \
 	test -f ${1} || (echo validation schema definition not found: ${SCHEMA_DEFINITION_FILE}; exit 1) && \
-	node merge-config.js $(CONFIG_BASE_FILE) domains/${DOMAIN}/${SCHEMA_ENVIRONMENT}.config.json $(CONFIG_TEST_FILE) && \
+	node merge-config.js $(CONFIG_BASE_FILE) domains/${DOMAIN}/${ENVIRONMENT}.config.json $(CONFIG_TEST_FILE) && \
```

Comment:
Is there a risk here that some commands will stop working if you don't supply either acceptance or production as the environment? Given that the default environment is development, for which there is not config.json available.

Reply:
The script will error when the corresponding config file cannot be found. This won't be an issue in deployment, because there we're only using the values `acceptance` and `production`. Locally it could be issue, but nothing serious.